pkgs/by-name/in/intel-compute-runtime/package.nix

```diff
@@ -12,13 +12,13 @@
 
 stdenv.mkDerivation rec {
   pname = "intel-compute-runtime";
-  version = "25.22.33944.8";
+  version = "25.27.34303.6";
```

Comment:
Their "Releases" page points to "25.27.34303.5" however I do see:

https://github.com/intel/compute-runtime/tree/25.27.34303.6
and even `25.27.34303.9`:
https://github.com/intel/compute-runtime/tree/25.27.34303.9

I don't mind keeping it `25.27.34303.6` or updating it to `25.27.34303.9`. Just wanted to point out ;)

Reply:
We only want the latest release. There are usually pre releases or other tags which are ignored.